recipes-omnect/initrdscripts/omnect-os-initramfs/common-sh

```diff
@@ -48,70 +48,43 @@ run_cmd_stdout() {
 
 # check ext4 filesystem
 # re-format partition in the case of uncorrectable fsck errors or enforced flag
-check_ext4_fs()
+check_fs()
 {
-    local   ext4_label=$1  # filesystem label
-    local block_device=$2  # block device name
-    local enforce_mkfs=$3  # enforce reformatting: 0|1
+    local label=$1  # label
     local fsck_res=0
     local kmsg_ratelimit=$(cat /proc/sys/kernel/printk_devkmsg)
 
-    msg "check_ext4_fs ${ext4_label} ${block_device}"
+    msg "check_fs /dev/omnect/${label}"
     echo "on" > /proc/sys/kernel/printk_devkmsg
 
-    if [ -z "${block_device}" ] || [ ! -b "${block_device}" ]; then
-        msg_fatal "\"${block_device}\" not a valid block device"
-        return 1
-    fi
```

Comment:
block_device no longer exists, but wouldn't it make sense to check for /dev/omnect/$label instead?

Reply:
imho, if `10-rootblk_dev` failed, we're doomed anyway